packages/cloud-cognitive/src/components/Toolbar/_toolbar.scss

```diff
@@ -5,38 +5,55 @@
 // LICENSE file in the root directory of this source tree.
 //
 
-@import '../../global/styles/project-settings';
-
 @import 'carbon-components/scss/components/button/button';
 
+@import '../../global/styles/project-settings';
+
 @include exports($name: cloud-cognitive--toolbar) {
   $block-class: #{$pkg-prefix}--toolbar;
+
   $border: carbon--rem(
       $px: 1px,
     )
     solid $ui-03;
 
+  $dimensions: carbon--rem(
+    $px: 40px,
```

Comment:
I believe you could just use `$spacing-08` here

Reply:
Thanks — would we typically use spacing tokens for properties other than [margin and padding](https://carbondesignsystem.com/guidelines/spacing/overview/#applying-spacing)? As I understand it, Carbon mostly favors [`rem` for height and width dimensions](https://github.com/search?q=%27width%3A%27+path%3Apackages%2Fstyles+repo%3Acarbon-design-system%2Fcarbon&type=Code). What's the current thinking here?

Reply:
You’re totally right! Nevermind then, what you had is good